ui/ozone/platform/wayland/wayland_pointer.h

```diff
@@ -57,12 +57,17 @@ class WaylandPointer {
 
   void SetSerial(uint32_t serial);
 
+  int GetFlagsWithKeyboardModifiers();
```

Comment:
maybe GetKeyboardModifierFlags?

Reply:
hum, ok. maybe not. flags can contain keyboard modifiers and more flags, it seems.